src/NServiceBus.AcceptanceTests/Causation/When_a_message_is_sent.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace NServiceBus.AcceptanceTests.Causation
+{
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_a_message_is_sent : NServiceBusAcceptanceTest
+    {
+
+        [Test]
+        public void Should_flow_causation_headers()
```

Comment:
We actually call those headers "causation headers" since correlation is another concept. That said I wonder if correlation should also be under the "causation" umbrella?